### DISCUSSION TITLE
Finetune specific coco-classes

### DISCUSSION THREAD

**Discussion by Rizhiy** [facebookresearch/detectron2#4848] [Q&A]
Hi, I would like to finetune a model to better recognise people, cats and dogs.
I have some data with those three classes labeled, but it is not a lot. Therefore I would like to preserve existing weights for those classes from a pre-trained COCO model.

Is there already an existing functionality for such task?

The way I see it, only the classification head needs to be modified. Rows/Columns for required classes should be identified and the rest can be dropped.

**Comment by 3i-Haseeb** [comment]
@Rizhiy Hi. Have you been able to do so? I want to improve performance on `dog` and `horse` categories by fine-tuning on custom data. but results have been poor so far. It confuses `horse` as a `dog` and vice versa.
